src/main/java/org/elasticsearch/rest/action/admin/indices/analyze/RestAnalyzeAction.java

```diff
@@ -47,21 +58,72 @@ public RestAnalyzeAction(Settings settings, RestController controller, Client cl
     @Override
     public void handleRequest(final RestRequest request, final RestChannel channel, final Client client) {
         String text = request.param("text");
+        AnalyzeRequest analyzeRequest = new AnalyzeRequest(request.param("index"), text);
```

Comment:
This means that the body's text overrides the text in the query string. On the other hand, the rest of the parameters do differently and query_string input overrides the body one. Can we be consistent?